service/awsconfig/v4/resource/cloudformation/main_stack.go

```diff
@@ -72,7 +68,12 @@ func (r *Resource) getMainGuestTemplateBody(customObject v1alpha1.AWSConfig) (st
 		return "", microerror.Mask(err)
 	}
 
-	return r.getMainTemplateBody(cloudFormationGuestTemplatesDirectory, adp)
+	rendered, err := templates.Render(key.CloudFormationGuestTemplates(), adp)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	return rendered, nil
```

Comment:
Having the errors handled in every case makes the stack trace we get from the masking way more useful than hiding steps in between.